setup.py

```diff
@@ -37,9 +37,6 @@ def finalize_options(self):
 
 
 extensions = [
-    Extension('autosklearn.data.competition_c_functions',
-              sources=['autosklearn/data/competition_c_functions.pyx'],
-              language='c')
```

Comment:
Could you please drop the whole array?

Also, you can drop the `cmdclass` and `ext_modules` arguments to `setup()` and everything related to `install_requires`.